SU2_CFD/src/iteration_structure.cpp

```diff
@@ -2303,9 +2303,9 @@ void CDiscAdjFluidIteration::InitializeAdjoint(CSolver *****solver, CGeometry **
   bool heat = config[iZone]->GetWeakly_Coupled_Heat();
   bool interface_boundary = (config[iZone]->GetnMarker_Fluid_Load() > 0);
 
-  /*--- Initialize the adjoint of the objective function (typically with 1.0) ---*/
+//  /*--- Initialize the adjoint of the objective function (typically with 1.0) ---*/
```

Comment:
why did this get commented?

Reply:
Just because it's in an iteration class whereas it should rather be done on the driver level. Though that's the case anyway (might have been just done redundantly in the past) and I'm almost sure it can be removed. So it's commented out for a while if that makes sense.. O:) Can you confirm, @talbring ?